frontend/src/app/patient.schema.json

```diff
@@ -25,6 +25,10 @@
       "description": "",
       "type": "string"
     },
+    "patientLookupID": {
```

Comment:
Since we stole the `patientId` I assume we'll want to save the `lookupId` for the sake of the UI?

Hmmm, I think I may be missing a `createMappingInterface` entry though.